ext/search_kit/CRM/Search/Upgrader.php

```diff
@@ -139,9 +139,39 @@ public function upgrade_1003() {
    * @return bool
    */
   public function upgrade_1004() {
-    $this->ctx->log->info('Applying update 1000 - fix menu permission.');
+    $this->ctx->log->info('Applying update 1004 - fix menu permission.');
     CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET permission = 'administer CiviCRM data' WHERE url = 'civicrm/admin/search'");
     return TRUE;
   }
 
+  /**
+   * Upgrade 1005 - add acl_bypass column.
+   * @return bool
+   */
+  public function upgrade_1005() {
```

Comment:
So this feels right but just making a point that these Extension upgrades are only run via the Extension Upgrade process not the main upgrade process but it should flag that they need to be done anyway.

Reply:
IMO we ought to run extension upgrades as part of the main upgrader.